applications/GeoMechanicsApplication/custom_processes/apply_normal_load_table_process.h

```diff
@@ -0,0 +1,66 @@
+// KRATOS___
+//     //   ) )
+//    //         ___      ___
+//   //  ____  //___) ) //   ) )
+//  //    / / //       //   / /
+// ((____/ / ((____   ((___/ /  MECHANICS
+//
+//  License:         geo_mechanics_application/license.txt
+//
+//  Main authors:    Richard Faasse,
+//                   Anne van de Graaf,
+//                   Gennady Markelov
+//
+
+#pragma once
+
+#include "processes/process.h"
+
+//#include <memory>
+//#include <vector>
+
+namespace Kratos
+{
+
+class ModelPart;
+class Parameters;
+
+
+class KRATOS_API(GEO_MECHANICS_APPLICATION) ApplyNormalLoadTableProcess : public Process
+{
+public:
+    KRATOS_CLASS_POINTER_DEFINITION(ApplyNormalLoadTableProcess);
+
+    ApplyNormalLoadTableProcess(ModelPart&        rModelPart,
+                                const Parameters& rProcessSettings);
+
+    ~ApplyNormalLoadTableProcess() override;
+
+    ApplyNormalLoadTableProcess(const ApplyNormalLoadTableProcess&) = delete;
+    ApplyNormalLoadTableProcess& operator=(const ApplyNormalLoadTableProcess&) = delete;
+
+    void ExecuteInitialize() override;
+    void ExecuteInitializeSolutionStep() override;
+
```

Comment:
Recently, we have added an override for member function `Info()` to class `ApplyScalarConstraintTableProcess`, which comes in really handy during debugging. I would suggest you to add a similar override here too.

Reply:
Thank you. Info() is added now.